.github/workflows/building.yml

```diff
@@ -0,0 +1,47 @@
+name: Building
+
+on:
+  push:
+    branches: [ kawaii ]
+  pull_request:
+    branches: [ kawaii ]
+
+jobs:
+  build-windows:
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+        node-version: [ 14.x ]
+
+    steps:
+    - name: Check out
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Run npm
+      run: |
+        npm ci
+        npm install nexe
+
+    - name: Build Hexa
+      run: |
+        node bootstrap.js hexa.json
+        ./node_modules/.bin/nexe hexa-node.js --target windows-x64-14.15.3 --name hexa -o hexa
```

Comment:
@acidicMercury8 is there better than 14.x node?

Reply:
@PeyTy official Node.js action uses LTS releases. We can use 16.x as a maximum

Reply:
@PeyTy 14.x version is available as a maximum